course/layouts/partials/youtube_player.html

```diff
@@ -9,6 +7,6 @@
     width="640" height="385"
     data-setup='{
     "techOrder": ["youtube"],
-    "sources": [{"type": "video/youtube", "src": "https://www.youtube.com/watch?v={{ $youtubeKey }}"}]}'
+    "sources": [{"type": "video/youtube", "src": "https://www.youtube.com/watch?v={{ . }}"}]}'
```

Comment:
This sets up our `youtube_player.html` partial to accept a single string parameter that is the youtube ID, but we still need a file at `course/layouts/shortcodes/youtube.html` specifically to override the built in `youtube` shortcode to use our own with the following content:

```
{{ $youtubeId := index .Params 0 }}
{{ partial "youtube_player.html" $youtubeId }}
```

Reply:
I created it and forgot to add it to the commit 😄